lib/explorer/data_frame.ex

```diff
@@ -627,6 +630,7 @@ defmodule Explorer.DataFrame do
     * `:eol_delimiter` - A single character used to represent new lines. (default: `"\n"`)
     * `:backend` - The Explorer backend to use. Defaults to the value returned by `Explorer.Backend.get/0`.
     * `:lazy` - force the results into the lazy version of the current backend.
+    * `:encoding` - Encoding to use when reading the file. For now, the only possible values are `utf8` and `utf8-lossy`. The utf8-lossy option means that invalid utf8 values are replaced with � characters. (default: `"utf8"`)
```

Comment:
@philss Should we make them atoms instead of regular strings?

Reply:
@josevalim yeah, I think so! 

@Jhonatannunessilva there is an example of what we want to do in https://github.com/elixir-explorer/explorer/pull/770
But if you prefer, we can do in another PR as well :)

Reply:
No problem, I can do it in this PR